kpi/004.adoc

```diff
@@ -140,8 +136,9 @@ update frequency, `gmd:MD_MaintenanceFrequencyCode` can be set to `asNeeded`:
 |1
 |===
 
-Total possible score: 5 (100%)
+*Total possible score: 5 (100%)*
```

Comment:
Lines 123, 132 and 135

I recommend replacing "... is defined" with "is included". 

This is more consistent with the language used in the WIS Manual. e.g. "Requirement 8.2.4: Each WIS discovery metadata record describing geographic data shall **include** the description of ..."

Reply:
Fixed globally.